server/network/aoprotocol.py

```diff
@@ -664,7 +670,9 @@ def net_cmd_ms(self, args):
             self.ArgType.INT, # 24 # additive
             self.ArgType.STR, # 25 # effect
             self.ArgType.INT, # 26 # third_charid
-            self.ArgType.STR_OR_EMPTY, # 27 # video
+            self.ArgType.STR, # 27 # blipname
+            self.ArgType.INT, # 28 # slide
+            self.ArgType.STR_OR_EMPTY, # 29 # video
```

Comment:
This will actually break validation cuz Video HAS to be on position 27 :(
The validation monstrosity has caught up with us making this a difficult PR to merge